### PR TITLE
Fixed lr_mult for generator

### LINKED ISSUE #18: Incorrect lr_mult value when using find_generator
When using `find_generator`:

https://github.com/surmenok/keras_lr_finder/blob/7870043d86101010b2a644df22b4018af58534fb/keras_lr_finder/lr_finder.py#L76

`lr_mult` causes the learning rate to converge at `end_lr` after only 1 epoch. If you want to use 2 or more epochs, then your learning rate will exceed `end_lr` and end up training on a learning rate that is far too high than intended.

The fix here is (when `end_lr=1,start_lr=0.001,steps_per_epoch=1000,epochs=4`):

```python
lr_mult = ((float(end_lr) / float(start_lr)) ** (1. / float(steps_per_epoch*epochs)))
```
Here's a sample script to show how the two functions diverge `while cur_lr <= end_lr`
Figure_1

### PR DESCRIPTION
**Pull request by JonnoFTW**
Fixed #18

### CODE REVIEW

File: keras_lr_finder/lr_finder.py [hunk 1]
```diff
@@ -39,7 +40,7 @@ def on_batch_end(self, batch, logs):
 
     def find(self, x_train, y_train, start_lr, end_lr, batch_size=64, epochs=1):
         num_batches = epochs * x_train.shape[0] / batch_size
-        self.lr_mult = (float(end_lr) / float(start_lr)) ** (float(1) / float(num_batches))
+        self.lr_mult = (float(end_lr) / float(start_lr)) ** (1.0 / float(num_batches)*float(epochs))
```
Comment (surmenok) [inline keras_lr_finder/lr_finder.py]: Maybe some parentheses are missing? Now it's dividing 1 by num_batches and then multiplying by epochs, which is effectively dividing by (num_batches / epochs). I think the last "*" should be replaced with "/", or parentheses added.

**Review by surmenok — CHANGES_REQUESTED** [review]

File: keras_lr_finder/lr_finder.py (hunk 1 above)
Comment (JonnoFTW) [reply keras_lr_finder/lr_finder.py]: I changed the wrong function, my bad. My fix should only be concerned with the generator method, not the regular one.

**Review by surmenok — APPROVED** [review]
LGTM
